kong/plugins/jwt/handler.lua

```diff
@@ -150,6 +153,15 @@ local function unauthorized(message, www_auth_content, errors)
   return { status = 401, message = message, headers = { ["WWW-Authenticate"] = www_auth_content }, errors = errors }
 end
 
+-- set header keys from claims
+local function set_headers(conf, claims)
```

Comment:
If you rename the function to `set_headers_from_claims`, the comment is no longer needed.